tests/python/gaia-ui-tests/gaiatest/gaia_graphics_test.py

```diff
@@ -48,14 +48,18 @@ def tearDown(self):
         """At the end of test execution, it checks for the errors"""
         self.assertTrue(self.test_passed, msg=self.failcomment)
 
-    def take_screenshot(self):
+    def take_screenshot(self, browser=False, browserInstance=None):
```

Comment:
If I understand correctly the issue here is that the method takes us to the system frame and returns us to the active application frame. Therefore, if we're in the browser content frame when we call this method we'll be in the browser app (chrome) frame when the method returns. If that's the issue, I think this is the wrong fix - it prevents proper status bar detection, and adds complexity.

A better solution might be to implement a `take_screenshot` within your test case class that takes the browser app object as an argument and a boolean to indicate if we are expecting to be in the content. This could then switch to the content at the end of the method:

```python
def take_screenshot(browser, switch_to_content=False):
    GaiaTestCase.take_screenshot()
    if switch_to_content:
        browser.switch_to_content()
```